### DISCUSSION TITLE
Making an input field show a specific index of a registered array

### DISCUSSION THREAD

**Discussion by nfuaufagf** [react-hook-form/react-hook-form#6603] [Q&A]
I have an input field that registers to an array, but I only want it to show a specific index of that array. An example could be a quiz form, where there are multiple questions and the user has to use that field to register the answers. It would all be registered to an array, but the field will show index 0 for question 1, and index 1 for question 2 etc. Is there any way to achieve this?

**Comment by bluebill1049** [comment]
you probably have to use `watch` or `useWatch`.
